packages/quantic/force-app/main/default/lwc/quanticUtils/quanticUtils.js

```diff
@@ -688,3 +690,65 @@ export function isParentOf(element, targetElement) {
     false
   );
 }
+
+/**
+ * Copies text to clipboard using the Clipboard API.
+ * https://developer.mozilla.org/en-US/docs/Web/API/Clipboard
+ * @param {string} text 
+ */
+export async function copyToClipboard(text) {
+  if (navigator?.clipboard?.writeText) {
+    try {
+      await navigator.clipboard.writeText(text);
+    } catch (err) {
+      copyToClipboardFallback(text);
+    }
+  } else {
+    copyToClipboardFallback(text);
+  }
+}
+
+/**
+ * Copies text to clipboard using the DOM. 
+ * @param {string} text
+ */
+export function copyToClipboardFallback(text) {
+  const el = document.createElement('textarea');
+  el.value = text;
+  document.body.appendChild(el);
+  el.select();
```

Comment:
Would this take away the current focus of the user?

let's say I have my cursor in the searchbox for example, would calling this remove the current place where the focus is?

If that is the case, I would argue against using that fallback for accessibility reasons to not mess up with the user's focus.

Reply:
Very nice point! thank you.
I'll test that to exactly know that behaviour occurring, but at this point I'm really considering the removal of the fallback method since it's also marked as deprecated.